packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -100,22 +105,357 @@ class VMInfoList extends StatelessWidget {
       ],
     );
   }
+}
 
-  List<Widget> _prettyRows(BuildContext context, List<Row> rows) {
-    return [
-      for (int i = 0; i < rows.length; ++i)
-        _buildAlternatingRow(context, i, rows[i]),
-    ];
+List<Widget> _prettyRows(BuildContext context, List<Row> rows) {
+  return [
+    for (int i = 0; i < rows.length; ++i)
+      _buildAlternatingRow(context, i, rows[i]),
+  ];
+}
+
+Widget _buildAlternatingRow(BuildContext context, int index, Widget row) {
+  return Container(
+    color: alternatingColorForIndex(index, Theme.of(context).colorScheme),
+    height: defaultRowHeight,
+    padding: const EdgeInsets.symmetric(
+      horizontal: defaultSpacing,
+    ),
+    child: row,
+  );
+}
+
+/// A Refresh ToolbarAction button.
+class ToolbarRefresh extends ToolbarAction {
+  const ToolbarRefresh({
```

Comment:
either move this to `common_widgets.dart` or use the existing `RefreshButton` class from common_widgets.dart

Reply:
Moving it to common_widgets. The RefreshButton has a slightly different shape, and it has space for a label, even if it is an empty string.